### PR TITLE
Add elasticsearch-administer

### PR DESCRIPTION
**Pull request by esoergel**
@benrudolph @dannyroberts

### CODE REVIEW

File: scripts/elasticsearch-administer.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    cluster = ClusterClient(es)
+    print "\nCLUSTER HEALTH"
+    pprint(cluster.health())
+    print "\nPENDING TASKS"
+    pprint(cluster.pending_tasks())
+    print "\nNODES"
+    for node in get_nodes_info(es):
+        print node.name, node.docs
+    print "\nSHARD ALLOCATION"
+    cat = CatClient(es)
```
Comment (benrudolph) [inline scripts/elasticsearch-administer.py]: :cat: :smirk_cat:

**Comment by benrudolph** [comment]
this might make a good fab task too. fabric has a lot of the functionality of this built in (arg parsing, command naming, color printing, etc). not a blocker though

**Comment by dannyroberts** [comment]
Looks great to me! I'll merge as is, feel free to entertain @benrudolph's suggestion later if it resonates with you.

**Comment by esoergel** [comment]
Interesting idea, @benrudolph .  I think I'll leave as-is for now, but could be talked into moving it to fabric if/when this gets added to.  Are you thinking of this as part of `fab/fabfile.py` or a separate fabric script?  I like `argparse` because it's the standard python cmdline tool library and works well, but I can see how having more environment awareness could be helpful.

**Comment by benrudolph** [comment]
sweet, sounds good. yeah `fab/fabfile.py` though that thing is long overdue for a refactor into multiple files. yeah cause essentially you take out the need to specify a host_url since you can just go `fab production my_es_command` and you'll know where all the machines are

**Pull request merged**